### PR TITLE
Tab Controller, not finished, with error

### PR DESCRIPTION
**Pull request by HenryJames4**

### CODE REVIEW

File: riverdata/App.js [hunk 1]
```diff
@@ -22,6 +23,7 @@ function App() {
         <Stack.Screen name="Site Gauges" component={RDSiteGaugesScreen} />
         <Stack.Screen name="Gauge Graph" component={RDGaugeGraphScreen} />
       </Stack.Navigator>
+      <Tabs />
```
Comment (austimkelly) [inline riverdata/App.js]: @HenryJames4 See https://reactnavigation.org/docs/nesting-navigators

It looks like having a Tab controller inside of a navigation stack is a no-no.
Here's the error I get when running `npm run ios`

```
ERROR  Error: Another navigator is already registered for this container. You likely have multiple navigators under a single "NavigationContainer" or "Screen". Make sure each navigator is under a separate "Screen" container. See https://reactnavigation.org/docs/nesting-navigators for a guide on nesting.

This error is located at:
    in BottomTabNavigator (created by Tabs)
    in Tabs (created by App)
    in EnsureSingleNavigator
    in BaseNavigationContainer
    in ThemeProvider
    in NavigationContainerInner (created by App)
    in App (created by ExpoRoot)
    in ExpoRoot
    in RCTView (created by View)
    in View (created by AppContainer)
    in RCTView (created by View)
    in View (created by AppContainer)
    in AppContainer
    in main(RootComponent)
```

**Review by austimkelly — CHANGES_REQUESTED** [review]
Please review nested navigation comment and adjust accordingly.

**Comment by HenryJames4** [comment]
Tab controller is now functional, it is set as a stack screen inside the navigation container. However, it does not display on "State Site", "Site Gauges", and "Gauge Graph". Still trying to figure out how to get these stacks of screens to persist under the tab controller.

**Review by JP0512 — APPROVED** [review]
@HenryJames4 I merged your change and tested them. They look good to me.

**Review by austimkelly — APPROVED** [review]
LGTM. Tested with @JP0512

**Pull request merged**